sdks/python/apache_beam/runners/direct/test_stream_impl.py

```diff
@@ -29,13 +29,20 @@
 
 import itertools
 
-import grpc
+try:
+  import grpc
+  from apache_beam.portability.api import beam_runner_api_pb2_grpc
+except:
+  grpc = None
+  beam_runner_api_pb2_grpc = None
+  # A workaround for some internal tests which are missing grpc dependencyy.
```

Comment:
maybe change "for some internal tests" to "directrunner users who would cannot depend on grpc"